src/api/public/apidocs-new/paths/source_project_name_project_file.yaml

```diff
@@ -0,0 +1,60 @@
+get:
+  summary: Read a project file
+  description: |
+    Read a project file
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/file_name.yaml'
+    - in: query
+      name: meta
+      required: false
+      schema:
+        type: integer
+      description: Set to 1 to include _meta files
+      example: 1
+    - in: query
+      name: rev
+      required: false
+      schema:
+        type: integer
+      description: Revision number
+  responses:
```

Comment:
I also miss an example of 403 error here. At least, there is one raised by the exception `Package::ReadSourceAccessError` when calling `validate_read_access_of_deleted_package` in [this line](https://github.com/openSUSE/open-build-service/blob/7e5cf6630d2e5202ca5dd4796c66010560b8b234/src/api/app/controllers/source_controller.rb#L295).